app/views/spots/index.html.erb

```diff
@@ -5,7 +13,7 @@
   <tbody>
     <% @spots.each do |spot| %>
       <tr>
-        <td> <%= link_to spot.name, spots_path(spot) %> </td>
+        <td><%= link_to spot.name, spot_path(spot.id) %></td>
```

Comment:
You don't need the `.id` here! `spot_path(spot)` should work just fine. (Leave it if you prefer, though!)